### DISCUSSION TITLE
example isnt working

### DISCUSSION THREAD

**Discussion by alopez66** [vasturiano/globe.gl#198] [Q&A]
Hello im new to programming. How would i open an example globe after downloading the files? Everytime i try opening one its  failes to load <script> with source“file:///d3-dsv”
 “file:///index-array-by” 
“file:///globe.gl”.

**Comment by vasturiano** [comment]
@alopez66 you won't be able to just open the html files directly in the browser straight from the file system. You need to load them using a web server.
